docs/README.md

```diff
@@ -29,13 +29,14 @@ access:
   type: 'ociRegistry'
   imageReference: 'ubuntu:18.0'
 EOF
-$ component-cli ca resources add ./resource.yaml
+$ component-cli ca resources add . ./resource.yaml
```

Comment:
maybe we should add a comment that explain why the `.` is needed where is point to?